stl/inc/vector

```diff
@@ -1617,7 +1617,7 @@ private:
         const size_type _Oldcapacity = capacity();
 
         if (_Oldcapacity > max_size() - _Oldcapacity / 2) {
-            return _Newsize; // geometric growth would overflow
+            return max_size(); // geometric growth would overflow

```

Comment:
Rather than call `max_size()` twice, can we make a new local `const auto _Max = max_size()` and use it in the `if` test and contained `return` statement?